### PR TITLE
Updated quickstart

### PR DESCRIPTION
**Pull request by immber**
## What does this PR do?
Updates the quickstart documentation to be slightly easier to follow. Renames "Running" section to "Setup"

## How do I test this PR?

View the markdown preview

### CODE REVIEW

File: docs/source/01-01-talk-quickstart.md [hunk 1]
```diff
@@ -178,32 +175,35 @@ You can now start the application by running:
...
-first admin account, and specify the domain whitelist for domains that are
-allowed to have the comment box on.
+and walk through the initial setup steps. 
+
+* First, enter your **Organization Name** and **Organization Contact Email**. This will appear in emails when inviting new team members.
+* Next, create your Admin user. You can specify an **Email Address**, **Username**, and **Password**
+* Finally, enter your list of **Permitted Domains**, read [here](talk/configuring-talk/#permitted-domains) about whitelisting domains 
 
-_During development, ensure you whitelist 127.0.0.1:3000 otherwise the
+>_During development, ensure you whitelist 127.0.0.1:3000 otherwise the
```
Comment (wyattjoh) [inline docs/source/01-01-talk-quickstart.md]: Was the `>` a mistake?

File: docs/source/01-01-talk-quickstart.md [hunk 2]
```diff
@@ -178,32 +175,35 @@ You can now start the application by running:
...
+With Talk running, you can now navigate to
 [http://127.0.0.1:3000/admin/install](http://127.0.0.1:3000/admin/install)
-and go through the admin installation. There you will be prompted to create your
-first admin account, and specify the domain whitelist for domains that are
-allowed to have the comment box on.
+and walk through the initial setup steps. 
+
+* First, enter your **Organization Name** and **Organization Contact Email**. This will appear in emails when inviting new team members.
+* Next, create your Admin user. You can specify an **Email Address**, **Username**, and **Password**
+* Finally, enter your list of **Permitted Domains**, read [here](talk/configuring-talk/#permitted-domains) about whitelisting domains
```
Comment (wyattjoh) [inline docs/source/01-01-talk-quickstart.md]: It's important to include a leading slash to ensure links are not broken.

```diff
- [here](talk/configuring-talk/#permitted-domains) about whitelisting domains
+ [here](/talk/configuring-talk/#permitted-domains) about whitelisting domains 
```

File: docs/source/01-01-talk-quickstart.md (hunk 1 above)
Comment (immber) [reply docs/source/01-01-talk-quickstart.md]: Nope, that was supposed to be a quote indent, but it's not showing up on the preview, will remove

File: docs/source/01-01-talk-quickstart.md (hunk 2 above)
Comment (immber) [reply docs/source/01-01-talk-quickstart.md]: gotcha, fixed

**Review by wyattjoh — APPROVED** [review]
Looks great! Made a quick fix!

**Pull request merged**